lib/block_view.go

```diff
@@ -4368,67 +4355,6 @@ func (bav *UtxoView) _connectBitcoinExchange(
 		minBitcoinBurnWork = 0
 	}
 
-	if checkMerkleProof {
```

Comment:
Add the following comment:

In the past, we used to check the merkle proof on all BitcoinExchange transactions. Now, however, this is
no longer needed because we can assume that all Bitcoin txns prior to the DeflationBombBlockHeight are
valid.